contrib/kind.sh

```diff
@@ -22,15 +24,13 @@ run_kubectl() {
 # is deleted directly with kind `kind delete cluster --name ovn`, it restarts the host.
 # The root cause is unknown, this also can not be reproduced in Ubuntu 20.04 or
 # with Fedora32 Cloud, but it does not happen if we clean first the ovn-kubernetes resources.
-delete()
-{
-  timeout 5 kubectl --kubeconfig ${HOME}/admin.conf delete namespace ovn-kubernetes || true
+delete() {
+  timeout 5 kubectl --kubeconfig "${KUBECONFIG}" delete namespace ovn-kubernetes || true
   sleep 5
-  kind delete cluster --name ${KIND_CLUSTER_NAME:-ovn}
```

Comment:
parse_args calls delete directly, so the variable is not defaulted :smile: , we need to keep the defaulting here too
It's a bit gross but this was only a bug in a specific fedora version, so I think we can live with that XD

Reply:
sure